apps/api/src/app/events/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -42,14 +42,6 @@ export class TriggerEvent {
       },
     });
 
-    this.logger.assign({
-      transactionId: command.transactionId,
-      environmentId: command.environmentId,
-      organizationId: command.organizationId,
-    });
```

Comment:
Don't we still want to have this assigned here to be available on all the scope logs?